COLLABORATOR_GUIDE.md

```diff
@@ -444,6 +444,10 @@ commit logs, ensure that they are properly formatted, and add
 * The commit message text must conform to the
 [commit message guidelines](./CONTRIBUTING.md#step-3-commit).
 
+Run tests (`make test` or `vcbuild test`). Even though there was a successful
```

Comment:
+1, I think we standardised on `-j4`, so it should probably be `make -j4 test`. I know it doesn't make the tests any faster, but if a rebuild is necessary it will help with that.